client/layout/index.jsx

```diff
@@ -49,9 +49,12 @@ import { handleScroll } from './utils';
 import 'calypso/components/environment-badge/style.scss';
 import './style.scss';
 
-function SidebarScrollSynchronizer() {
+function SidebarScrollSynchronizer( { layoutFocus } ) {
 	const isNarrow = useBreakpoint( '<660px' );
-	const active = ! isNarrow && ! config.isEnabled( 'jetpack-cloud' ); // Jetpack cloud hasn't yet aligned with WPCOM.
+	const active =
+		! isNarrow &&
+		! config.isEnabled( 'jetpack-cloud' ) && // Jetpack cloud hasn't yet aligned with WPCOM.
+		layoutFocus !== 'sites';
```

Comment:
@danielbachhuber do you about why this `SidebarScrollSynchronizer` component needed to be deactivated when in the `sites` layout focus? I removed this line and everything worked as I expected.

Reply:
> do you about why this `SidebarScrollSynchronizer` component needed to be deactivated when in the `sites` layout focus? I removed this line and everything worked as I expected.

I don't, no. Everything works as expected when I remove it too.

I've reverted it with a1085d287b